src/insights_client/__init__.py

```diff
@@ -237,9 +237,14 @@ def _main():
             sys.exit('Unable to load Insights Config')
 
         if config["version"]:
-            from insights_client.constants import InsightsConstants as constants
-            print("Client: %s" % constants.version)
-            print("Core: %s" % InsightsClient().version())
+            try:
+                from insights_client.constants import InsightsConstants
+            except ImportError:
+                # The source file is build from 'constants.py.in' and is not available during development
+                class InsightsConstants(object):
+                    version = "upstream"
+            print("Client: %s" % InsightsConstants.version)
+            print("Core:   %s" % InsightsClient().version())
```

Comment:
I'd avoid changing the formatting of this output; for weird as it is, there might be something somewhere parsing this...